source/reference/api/start.txt

```diff
@@ -162,6 +162,62 @@ Request Body Parameters
        .. include:: /includes/api/facts/namespace-explanation.rst
 
        .. versionadded:: 1.6
+   * - ``namespaceRemap``
+     - array
+     - Optional
+     - Array of documents that specify namespace changes to make during sync.
+
+       .. versionadded:: 1.8 Beta
+
+   * - ``namespaceRemap[n].`` ``from``
+     - document
+     - Required
+     - Document that specifies the database and collection on the source cluster
+       to read from in the remapping operation.
+
+       .. versionadded:: 1.8 Beta
+
+   * - ``namespaceRemap[n].`` ``from.database``
+     - string
+     - Required
+     - Database to read from the source cluster.
+
+       .. versionadded:: 1.8 Beta
+
+   * - ``namespaceRemap[n].`` ``from.collection``
+     - string
+     - Conditional
+     - Collection to read from the source cluster. If the ``collection`` field
+       is present in the ``from`` document, it must also occur in the ``to``
+       document. If it's absent from both documents, ``mongosync`` remaps all
+       collections in database.
+
+       .. versionadded:: 1.8 Beta
+
+   * - ``namespaceRemap[n].`` ``to``
+     - document
+     - Required
+     - Document that specifies the database and collection on the destination cluster
+       to write to in the remapping operation.
+
+       .. versionadded:: 1.8 Beta
+
+   * - ``namespaceRemap[n].`` ``to.database``
+     - string
+     - Required
+     - Database to write to the destination cluster.
```

Comment:
[suggestion]

To match the language in `to.collection`
```suggestion
     - Database to write to in the destination cluster.
```

Reply:
Fixed